pallets/collator-assignment/src/lib.rs

```diff
@@ -159,16 +167,130 @@ pub mod pallet {
     }
 
     impl<T: Config> Pallet<T> {
+        pub(crate) fn enough_collators_for_all_chains(
+            bulk_paras: &Vec<ChainNumCollators>,
+            pool_paras: &Vec<ChainNumCollators>,
```

Comment:
Clippy will complain

`&[ChainNumCollators]`

Reply:
It didn't. I think because `bulk_paras` and `pool_paras` are vector already.